app/src/main/java/ch/epfl/culturequest/social/PictureAdapter.java

```diff
@@ -125,11 +125,32 @@ public void onBindViewHolder(@NonNull PictureViewHolder holder, int position) {
 
         holder.location.setText("Lausanne");
 
+        // Set the like count
+        holder.likeCount.setText(getNumberOfLikes(post.getLikes()));
```

Comment:
I am just thinking of what the like count looks like for other users. Do you feel that it is weird to have a like count with the word "like" right next to the heart?

Reply:
I thought about it. I think either way is fine, but it still made more sense to me to have "likes" next to it.